content/docs/hello-world.md

```diff
@@ -15,36 +15,33 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+在頁面上顯示「Hello, world!」。
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+點擊上方連結開啟線上編輯器。請隨意地將程式碼做任何的改變，並觀察它們的輸出變化。大部分的文件頁面都有像這樣可以編輯的範例。
```

Comment:
nit: I feel like we usually speak in active tone rather than passive tone much more than English does. "請隨意的將程式碼做任何的改變，並觀察它們的輸出變化。" -> "請隨意的變更程式碼並觀察它們的輸出變化。" feels more smoother for me. :smile: